optaplanner-examples/src/main/java/org/optaplanner/examples/flightcrewscheduling/solver/move/FlightCrewSequentialPillarSwapMove.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.flightcrewscheduling.solver.move;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.optaplanner.core.impl.heuristic.move.AbstractMove;
+import org.optaplanner.core.impl.score.director.ScoreDirector;
+import org.optaplanner.examples.flightcrewscheduling.domain.Employee;
+import org.optaplanner.examples.flightcrewscheduling.domain.FlightAssignment;
+import org.optaplanner.examples.flightcrewscheduling.domain.FlightCrewSolution;
+
+public class FlightCrewSequentialPillarSwapMove extends AbstractMove<FlightCrewSolution> {
+
+    private final Employee leftEmployee;
+    private final List<FlightAssignment> leftFlightAssignmentList;
+    private final Employee rightEmployee;
+    private final List<FlightAssignment> rightFlightAssignmentList;
+
+    public FlightCrewSequentialPillarSwapMove(Employee leftEmployee, List<FlightAssignment> leftFlightAssignmentList,
+            Employee rightEmployee, List<FlightAssignment> rightFlightAssignmentList) {
+        this.leftEmployee = leftEmployee;
+        this.leftFlightAssignmentList = leftFlightAssignmentList;
+        this.rightEmployee = rightEmployee;
+        this.rightFlightAssignmentList = rightFlightAssignmentList;
+    }
+
+    @Override
+    public boolean isMoveDoable(ScoreDirector<FlightCrewSolution> scoreDirector) {
+        return true;
```

Comment:
Should be the move doable is both left and right `FlightAssignmentList` are empty?